### PR TITLE
Benefits of Speaking at DjangoCon Post

### PR DESCRIPTION
**Pull request by jcjudkins**
- Added my name to the contributor list.
- Wrote blog post about the benefits of being a presenter.

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.10
```
Comment (drewbrew) [inline .python-version:1]: We probably don't need to commit this (or should use 3.11)

File: _posts/2023-05-08-benefits-of-speaking-djangocon.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+ The DjangoCon US team would like to make sure you are aware of some of the benefits of giving a talk at DjangoCon US 2023.
+
+# Benefits you will receive for being a presenter:
+
+- **FREE! Ticket to DjancoCon US 2023.**
+
+- **Up to $500 in travel cost reimbursement!**
+
+- **A professionally produced video of your talk will be published on our YouTube channel.** (You may opt-out of this.)
```
Comment (drewbrew) [inline _posts/2023-05-08-benefits-of-speaking-djangocon.md:20]:
```suggestion
- **A professionally produced video of your talk will be published on our YouTube channel.** (You may opt out of this.)
```

You don't need the hyphen when using this as a verb

**Review by drewbrew — APPROVED** [review]
Thanks for writing this!

File: .python-version (hunk 1 above)
Comment (jcjudkins) [reply .python-version]: Removed!

**Comment by jcjudkins** [comment]
> We should probably just include the bio and blog files here and not the package-lock/static/etc files.

Removed all but the bio and blog files as recommended. Thank you!

**Pull request closed without merging**